logback-classic/src/test/java/ch/qos/logback/classic/pattern/ConverterTest.java

```diff
@@ -398,4 +402,23 @@ public void contextProperty() {
     String result = converter.convert(event);
     assertEquals("v", result);
   }
+
+  @Test
+  @Ignore
+  public void performanceTest() {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://www.codacy.com/app/tony19/logback/file/2371315592/issues/source?bid=3222618&fileBranchId=3222665#l408)